x-pack/plugins/security_solution/public/common/components/navigation/use_get_url_search.tsx

```diff
@@ -13,7 +13,7 @@ import { SearchNavTab } from './types';
 
 export const useGetUrlSearch = (tab: SearchNavTab) => {
   const mapState = makeMapStateToProps();
-  const { urlState } = useDeepEqualSelector(mapState);
+  const urlState = useDeepEqualSelector((state) => mapState(state).urlState);
```

Comment:
does it really matter to do that? I feel like that it does not because of the memoization from the makeMapStateToProps and also because we are returning urlSearch

Reply:
you're right 👍